src/sections/collection/datasets-list/file-card/FileCard.module.scss

```diff
@@ -0,0 +1,74 @@
+@use 'sass:color';
+@import "node_modules/@iqss/dataverse-design-system/src/lib/assets/styles/design-tokens/colors.module";
+@import "node_modules/@iqss/dataverse-design-system/src/lib/assets/styles/design-tokens/typography.module";
+
+.container {
+  margin: 6px 0;
+  padding: 4px 10px;
+  border: 1px solid $dv-info-border-color;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.title {
+  display: flex;
+
+  > * {
+    margin-right: .5em;
+  }
+}
+
+.icon {
+  margin-top: 2px;
+  font-size: 1.3em;
+  line-height: 1.1;
+
+  > div >span {
+    margin-right: 0;
+  }
+}
+
+.thumbnail {
+  width: 48px;
+  margin: 8px 12px 6px 0;
+  font-size: 2.8em;
+
+  img {
+      vertical-align: top;
+  }
+}
+
+.info {
+  display: flex;
+  color: $dv-subtext-color;
+}
+
+.description {
```

Comment:
The description tag could be a p (paragraph)  maybe?
I mean this one:
`<span className={styles.description}>{filePreview.metadata.description}</span>`

Also I noticed that description class is also used for the container that wraps everything inside FileCardInfo,  we should create another class with another name for the main container to avoid confusion.

One thing good to add here is after how much lines the text should show ellipsis (...) and cut it, so the FileCard doesn't get grow too much in height if the description is too long.

```
.description {
   ...rest of properties

   // The css trick 👇
   overflow: hidden;
   display: -webkit-box;
   -webkit-line-clamp: 3; /* number of lines to show, apply what you think looks better */
           line-clamp: 3; 
   -webkit-box-orient: vertical;
}
```
Will change this:

>         Lorem, ipsum dolor sit amet consectetur adipisicing elit. Aliquid repudiandae minus quod
>         ipsam, eum obcaecati recusandae accusamus repellendus quas quo! Recusandae alias nam quidem
>         fugiat commodi est quaerat maiores aperiam odio nesciunt nemo natus in perferendis, earum
>         veritatis dolores nulla error quasi non aliquam. Odit, ratione autem sapiente ea velit
>         nesciunt obcaecati nostrum cumque ut sint quam voluptatum quasi! Dicta.

        
To this:

>         Lorem, ipsum dolor sit amet consectetur adipisicing elit. Aliquid repudiandae minus quod
>         ipsam, eum obcaecati recusandae accusamus repellendus quas quo! Recusandae alias nam quidem
>         fugiat commodi est quaerat maiores aperiam odio nesciunt nemo natus in perferendis, earum...

Reply:
Thanks! I applied the CSS trick for overflow and it works well.  I tried changing the <span> to <p> but it didn't have much affect on the layout, is that the reason for the change? Or is it more semantic to use <p>?

Reply:
Yeah, exactly, it's only for semantic reasons 👍🏼

Reply:
sounds good, just changed it :)